javascript/javascript3/week2/main.js

```diff
@@ -0,0 +1,42 @@
+/* This code will move the li to the position 100, 100. Calling moveElement will return
+a promise that resolves after the li element has been moved.*/
+
+const redBox = document.querySelector("ul.marks li:nth-child(1)");
+const blueBox = document.querySelector("ul.marks li:nth-child(2)");
+const greenBox = document.querySelector("ul.marks li:nth-child(3)");
+const boxes = [redBox, blueBox, greenBox];
+
+function moveElement(boxToMove, newPosition) {
+  return new Promise((resolve) => {
+    boxToMove.style.transform = `translate(${newPosition.x}px, ${newPosition.y}px)`;
+    boxToMove.addEventListener("transitionend", resolve);
+  });
+}
+
```

Comment:
Remove these lines. 
You don't need to re-declare the variables and the function. They already exist at `move-element.js` file
and at your `index.html` file you include it
```html
    <script src="move-element.js"></script>
    <script src="main.js"></script>
```

Reply:
done!